packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -64,6 +64,7 @@ export interface GCFLogger {
   warn: LogFn;
   error: LogFn;
   metric: LogFn;
+  flushSync: Function;
```

Comment:
Can we explicitly declare the param signature here?

Reply:
sure thing, I'll add that